python/lsst/ts/electrometer/__init__.py

```diff
@@ -19,8 +19,11 @@
 # You should have received a copy of the GNU General Public License
 # along with this program.  If not, see <https://www.gnu.org/licenses/>.
 
-from . import csc
-from . import controller
+from .csc import *
```

Comment:
I think wildcard imports aren't PEP8 compliant?

Reply:
In this case, the __init__.py is fine because it is adding to the package namespace. Meaning that one can do `from lsst.ts import electrometer` and get all the underlying classes.